diff -r 0c4b60ac68b4 src/wmtrace_resource.erl
--- a/src/wmtrace_resource.erl	Tue Jun 15 08:36:55 2010 -0400
+++ b/src/wmtrace_resource.erl	Mon May 23 12:31:10 2011 +0100
@@ -5,8 +5,7 @@
 -export([add_dispatch_rule/2,
          remove_dispatch_rules/0]).
 
--export([start_link/1,
-         ping/2,
+-export([ping/2,
          init/1,
          resource_exists/2,
          content_types_provided/2,
@@ -52,9 +51,6 @@
 %% Resource
 %%
 
-start_link(Args) ->
-    webmachine_resource:start_link(?MODULE, [Args]).
-
 ping(ReqData, State) ->
     {pong, ReqData, State}.
 
